libs/application/templates/aosh/seminars/src/fields/Participants/index.tsx

```
@@ -0,0 +1,387 @@
+                parsedParticipant.email &&
+                allEmails.includes(parsedParticipant.email)
+              if (isEmailDuplicate) {
+                allEmailWarnings.push(index + 1)
+                return validParticipants
+              }
+
+              // Add email to tracking array
+              parsedParticipant.email && allEmails.push(parsedParticipant.email)
+
+              // Add valid participant to results
+              return [...validParticipants, parsedParticipant]
```

Biome (1.9.4)

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)